src/FQDN.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Etsy\Opsweekly;
+
+use \InvalidArgumentException;
```

Comment:
not sure why this is included. Is this thrown somewhere implicitly? Should it be caught somewhere as well then?